File: dev/ProyectoCadima/Entidades/Seguridad/Producto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entidades.Seguridad
{
    public class Producto
    {
        public int ProductoId { get; set; }
        public int TipoIdc { get; set; }
        public int EmpresaId { get; set; }
        public string Nombre { get; set; }
        public decimal Precio { get; set; }
        public bool Estado { get; set; }
        public string NombreEmp { get; set; }
        public Producto()
        {
        }
    }
}

File: dev/ProyectoCadima/android/app/src/main/java/com/nur/compraonline/model/security/Producto.java
package com.nur.compraonline.model.security;

import com.nur.compraonline.model.Entity;

/**
 * Created by dpedrazas on 12/09/2017.
 */

public class Producto extends Entity {

    private Integer ProductoId;
    private Integer TipoIdc ;
    private Integer EmpresaId ;
    private String Nombre ;
    private Double Precio ;
    private Boolean Estado ;

    public Producto() {
    }

    public Integer getProductoId() {
        return ProductoId;
    }

    public void setProductoId(Integer productoId) {
        ProductoId = productoId;
    }

    public Integer getTipoIdc() {
        return TipoIdc;
    }

    public void setTipoIdc(Integer tipoIdc) {
        TipoIdc = tipoIdc;
    }

    public Integer getEmpresaId() {
        return EmpresaId;
    }

    public void setEmpresaId(Integer empresaId) {
        EmpresaId = empresaId;
    }

    public String getNombre() {
        return Nombre;
    }

    public void setNombre(String nombre) {
        Nombre = nombre;
    }

    public Double getPrecio() {
        return Precio;
    }

    public void setPrecio(Double precio) {
        Precio = precio;
    }

    public Boolean getEstado() {
        return Estado;
    }

    public void setEstado(Boolean estado) {
        Estado = estado;
    }
}
